### PR TITLE
updated pricing faq

### PR DESCRIPTION
**Pull request by raghavjain3**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -37,13 +37,47 @@ Help is available for all users, free or otherwise, through the [r2c Community S
...
+#### Can I use Semgrep for free?
 
-See the Semgrep feature and pricing comparison at [r2c.dev/pricing](https://r2c.dev/pricing)
+Yes, you can use the Semgrep CLI and Semgrep Community Tier for [free](https://semgrep.dev/getting-started)!
 
-#### Who is a user and how do you count them?
+#### What are the differences between a free plan and the paid plans?
 
-For the Semgrep Team tier, a user is any project contributor whose pull request or code was scanned in the billing cycle. For example, if a project has 4 unique contributors who pushed code during the period, and Semgrep runs on all their pull requests, there are 4 users. If these unique contributors push to many projects, they will only be counted once; please feel free to scan often and widely. Semgrep does not charge on a line-of-code or per-project basis.
+Semgrep Community is great for individuals looking to quickly secure their public and private projects.
```
Comment (chmccreery) [inline docs/faq.md:46]: May want to add "and teams looking to explore and try out the product" here

File: docs/faq.md [hunk 2]
```diff
@@ -37,13 +37,47 @@ Help is available for all users, free or otherwise, through the [r2c Community S
...
+
+Semgrep Team and Enterprise are best for larger organizations that require dedicated support and advanced functionality.
+
+You can find more on our [pricing page](http://semgrep.dev/pricing).
+
+#### Who is a user?
```
Comment (chmccreery) [inline docs/faq.md:52]:
```suggestion
#### How many seats do I need?
```

File: docs/faq.md [hunk 3]
```diff
@@ -37,13 +37,47 @@ Help is available for all users, free or otherwise, through the [r2c Community S
...
+
+A user is any project contributor whose pull request or code is scanned during the billing period.
```
Comment (chmccreery) [inline docs/faq.md:54]:
```suggestion
One seat corresponds to one project contributor whose pull request or code is scanned during the billing period.
```

File: docs/faq.md [hunk 4]
```diff
@@ -37,13 +37,47 @@ Help is available for all users, free or otherwise, through the [r2c Community S
...
 
-#### Who is a user and how do you count them?
+#### What are the differences between a free plan and the paid plans?
 
-For the Semgrep Team tier, a user is any project contributor whose pull request or code was scanned in the billing cycle. For example, if a project has 4 unique contributors who pushed code during the period, and Semgrep runs on all their pull requests, there are 4 users. If these unique contributors push to many projects, they will only be counted once; please feel free to scan often and widely. Semgrep does not charge on a line-of-code or per-project basis.
+Semgrep Community is great for individuals looking to quickly secure their public and private projects.
+
+Semgrep Team and Enterprise are best for larger organizations that require dedicated support and advanced functionality.
+
+You can find more on our [pricing page](http://semgrep.dev/pricing).
```
Comment (chmccreery) [inline docs/faq.md:50]:
```suggestion
You can find more on our [pricing page](https://semgrep.dev/pricing).
```

File: docs/faq.md [hunk 5]
```diff
@@ -37,13 +37,47 @@ Help is available for all users, free or otherwise, through the [r2c Community S
...
+
+#### Who is a user?
+
+A user is any project contributor whose pull request or code is scanned during the billing period.
+
+For example, if a project has 4 unique contributors who push code during the billing period and Semgrep runs on all their pull requests, there are 4 users. If these unique contributors commit to many projects, they will only be counted once.
```
Comment (chmccreery) [inline docs/faq.md:56]:
```suggestion
For example, if a project has 4 unique contributors who push code during the billing period and Semgrep runs on all their pull requests, that is counted as 4 seats. If these unique contributors commit to many projects, they will only be counted once.
```

File: docs/faq.md [hunk 6]
```diff
@@ -37,13 +37,47 @@ Help is available for all users, free or otherwise, through the [r2c Community S
...
+
+To upgrade to our Team tier, select the number of seats you wish to purchase and pay directly from the Settings page in Semgrep App!
+
+For custom pricing on our Enterprise tier, please contact us via the form on our [pricing page](http://semgrep.dev/pricing).
+
+#### How do I pay for my plan?
+
+We accept all major credit cards through [Stripe](https://stripe.com/). If you want to pay through a purchase order / invoice mechanism, you can also reach out to [billing@r2c.dev](mailto:billing@r2c.dev) to get in touch with our sales team.
+
+We will bill you every 30 days.
```
Comment (chmccreery) [inline docs/faq.md:68]: Is this true? Is it every 30 days or every month?

**Review by chmccreery — APPROVED** [review]
Looks way better! There is a little bit of inconsistency between the terms "seat" and "user" and "developer" here, so I suggested some small modifications. Take them or leave them or change them 😄

File: docs/faq.md (hunk 6 above)
Comment (raghavjain3) [reply docs/faq.md:68]: fixed to say every month

File: docs/faq.md [hunk 7]
```diff
@@ -37,13 +37,47 @@ Help is available for all users, free or otherwise, through the [r2c Community S
...
 
-#### Who is a user and how do you count them?
+#### What are the differences between a free plan and the paid plans?
 
-For the Semgrep Team tier, a user is any project contributor whose pull request or code was scanned in the billing cycle. For example, if a project has 4 unique contributors who pushed code during the period, and Semgrep runs on all their pull requests, there are 4 users. If these unique contributors push to many projects, they will only be counted once; please feel free to scan often and widely. Semgrep does not charge on a line-of-code or per-project basis.
+Semgrep Community is great for individuals looking to quickly secure their public and private projects and teams looking to explore and try out the product.
+
+Semgrep Team and Enterprise are best for larger organizations that require dedicated support and advanced functionality.
+
+You can find more on our [pricing page](https://semgrep.dev/pricing).
```
Comment (pabloest) [inline docs/faq.md:50]:
```suggestion
You can find more on the [pricing page](https://semgrep.dev/pricing).
```

File: docs/faq.md [hunk 8]
```diff
@@ -37,13 +37,47 @@ Help is available for all users, free or otherwise, through the [r2c Community S
...
+
+#### How many seats do I need?
+
+One seat corresponds to one project contributor whose pull request or code is scanned during the billing period.
```
Comment (pabloest) [inline docs/faq.md:54]:
```suggestion
One seat corresponds to one project contributor whose merge/pull request or code is scanned during the billing period.
```

File: docs/faq.md [hunk 9]
```diff
@@ -37,13 +37,47 @@ Help is available for all users, free or otherwise, through the [r2c Community S
...
+
+For example, if a project has 4 unique contributors who push code during the billing period and Semgrep runs on all their pull requests, that is counted as 4 seats. If these unique contributors commit to many projects, they will only be counted once.
```
Comment (pabloest) [inline docs/faq.md:56]:
```suggestion
For example, if a project has 4 unique contributors who push code during the billing period and Semgrep runs on their merge/pull requests, 4 seats required. If these unique contributors commit to many projects within the same organization, they will only be counted once, so no additional seats are required.
```

Not sure if my edit about the same organization is correct or not?

File: docs/faq.md [hunk 10]
```diff
@@ -37,13 +37,47 @@ Help is available for all users, free or otherwise, through the [r2c Community S
...
+
+#### How do I upgrade my plan?
+
+To upgrade to our Team tier, select the number of seats you wish to purchase and pay directly from the Settings page in Semgrep App!
```
Comment (pabloest) [inline docs/faq.md:60]:
```suggestion
To upgrade to the Team tier, select the number of seats you wish to purchase and pay directly from the Settings page in Semgrep App!
```

File: docs/faq.md [hunk 11]
```diff
@@ -37,13 +37,47 @@ Help is available for all users, free or otherwise, through the [r2c Community S
...
+
+For custom pricing on our Enterprise tier, please contact us via the form on our [pricing page](http://semgrep.dev/pricing).
```
Comment (pabloest) [inline docs/faq.md:62]:
```suggestion
For Enterprise tier pricing, please contact us via the form on the [pricing page](http://semgrep.dev/pricing).
```

File: docs/faq.md [hunk 12]
```diff
@@ -53,7 +53,7 @@ You can find more on the [pricing page](https://semgrep.dev/pricing).
 
 One seat corresponds to one project contributor whose merge/pull request or code is scanned during the billing period.
 
-For example, if a project has 4 unique contributors who push code during the billing period and Semgrep runs on all their pull requests, that is counted as 4 seats. If these unique contributors commit to many projects, they will only be counted once.
+For example, if a project has 4 unique contributors who push code during the billing period and Semgrep runs on their merge/pull requests, 4 seats required. If these unique contributors commit to many projects within the same organization, they will only be counted once, so no additional seats are required.
```
Comment (pabloest) [inline docs/faq.md:56]: @raghavjain3 my mistake, this should be “…merge/pull requests, 4 seats are required.”

(missing the _are_)

**Pull request merged**